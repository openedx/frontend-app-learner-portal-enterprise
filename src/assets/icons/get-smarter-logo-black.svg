<svg width="182" height="47" viewBox="0 0 182 47" fill="none" xmlns="http://www.w3.org/2000/svg">
<g clip-path="url(#clip0_54_66849)">
<path d="M145.304 22.7131C144.067 22.7131 142.83 22.5072 141.645 21.9926C140.563 21.5294 139.584 20.8603 138.759 19.9854C137.935 19.1105 137.316 18.0811 136.853 16.9489C136.389 15.7137 136.183 14.427 136.183 13.1404C136.183 11.9052 136.389 10.67 136.853 9.48629C137.265 8.4055 137.883 7.37618 138.656 6.50125C139.429 5.62633 140.408 4.95727 141.439 4.44261C142.521 3.92794 143.758 3.67061 144.943 3.67061C146.231 3.67061 147.468 3.92794 148.602 4.44261C149.632 4.95727 150.56 5.62633 151.333 6.55272C152.054 7.47911 152.621 8.50843 152.982 9.58922C153.343 10.7729 153.549 11.9567 153.549 13.1919V14.3241H139.532C139.584 14.8902 139.738 15.4564 139.944 15.971C140.202 16.6916 140.614 17.3606 141.13 17.9267C141.645 18.4929 142.263 18.9046 142.933 19.2134C143.706 19.5222 144.531 19.6766 145.355 19.6766C146.283 19.6766 147.262 19.5222 148.138 19.1619C149.014 18.8017 149.787 18.2355 150.405 17.5665L150.921 17.0004L152.724 19.2134L152.312 19.6251C151.436 20.5515 150.405 21.272 149.22 21.7867C147.983 22.5072 146.644 22.7646 145.304 22.7131ZM131.751 22.7131C130.617 22.7646 129.535 22.3528 128.762 21.5294C128.092 20.6545 127.731 19.5222 127.783 18.3899V7.17031H125V4.23674H127.783V-0.34375H130.978V4.23674H134.379V7.17031H130.978V18.1841C130.978 18.6473 131.081 19.059 131.287 19.4707C131.442 19.7281 131.751 19.831 132.06 19.831C132.318 19.831 132.627 19.7795 132.885 19.6251C133.091 19.5222 133.297 19.4193 133.451 19.2134L134.121 18.4929L135.461 21.1691L135.1 21.5294C134.688 21.9411 134.224 22.2499 133.761 22.4558C133.091 22.6616 132.421 22.7646 131.751 22.7131ZM101.605 22.7131C100.78 22.7131 100.007 22.5587 99.2344 22.3014C98.4614 22.044 97.74 21.6323 97.1216 21.1176C96.5032 20.5515 95.9879 19.8825 95.6272 19.1619C95.2149 18.3385 95.0603 17.4121 95.0603 16.4857C95.0603 15.5593 95.2149 14.6329 95.6272 13.8095C95.9879 13.0375 96.5032 12.3684 97.1216 11.8537C97.74 11.3391 98.4614 10.9273 99.2344 10.67C100.007 10.4127 100.78 10.3098 101.605 10.3098C103.408 10.2583 105.212 10.8244 106.655 11.8537V9.94949C106.707 9.07456 106.294 8.19964 105.573 7.63351C104.748 7.06738 103.769 6.75858 102.739 6.75858C101.863 6.75858 100.986 6.91298 100.162 7.27324C99.3375 7.68497 98.5645 8.2511 97.9461 9.0231L97.4308 9.64069L95.5757 7.32471L95.9879 6.91298C96.8639 5.93512 97.8946 5.16313 99.0798 4.64847C100.317 4.13381 101.656 3.82501 102.996 3.87648C103.872 3.87648 104.748 3.97941 105.573 4.23674C106.346 4.4426 107.119 4.80287 107.789 5.31753C108.459 5.83219 108.922 6.50125 109.283 7.27324C109.644 8.14817 109.798 9.0231 109.798 9.94949V22.4558H106.655V21.3235C105.212 22.1984 103.408 22.7646 101.605 22.7131ZM58.3182 22.7131C56.9784 22.7131 55.6901 22.5072 54.4534 22.044C53.2681 21.5808 52.186 20.8089 51.3615 19.831L51.0007 19.4193L52.9589 17.0004L53.4743 17.6694C54.0411 18.3385 54.711 18.8531 55.484 19.2134C56.36 19.6766 57.3907 19.8825 58.3698 19.8825C59.4004 19.9339 60.431 19.6766 61.2555 19.059C61.8739 18.5958 62.1831 17.8753 62.1831 17.1548C62.2346 16.6916 62.0285 16.2284 61.6678 15.9196C61.1525 15.5078 60.5341 15.199 59.9157 15.0446C59.1427 14.7873 58.3182 14.5814 57.3907 14.3756C56.4631 14.1697 55.5355 13.9124 54.6595 13.5521C53.835 13.1919 53.062 12.6772 52.4436 12.0081C51.7737 11.2361 51.4645 10.2068 51.4645 9.17749C51.4645 7.68497 52.1344 6.29539 53.2681 5.369C53.8865 4.85433 54.608 4.49407 55.3809 4.23674C56.257 3.97941 57.1845 3.82501 58.1121 3.82501C59.4519 3.77354 60.8433 4.03088 62.08 4.64847C63.0592 5.11167 63.9352 5.72926 64.6566 6.55272L65.0174 6.96445L63.1107 9.38336L62.5954 8.66283C62.1316 8.04524 61.5132 7.58204 60.8433 7.27324C60.0188 6.91298 59.0912 6.70712 58.1636 6.70712C57.2361 6.65565 56.257 6.91298 55.484 7.42764C54.9687 7.83937 54.608 8.45697 54.6595 9.12603C54.608 9.53776 54.8141 9.94949 55.1748 10.2068C55.7417 10.5671 56.3085 10.8759 56.9784 11.0303C57.6999 11.2361 58.4728 11.4935 59.5035 11.6993C60.431 11.9052 61.3071 12.214 62.1831 12.6257C63.0076 13.0375 63.7806 13.6036 64.399 14.3241C65.0689 15.1476 65.3781 16.2284 65.3265 17.3092C65.3265 18.0811 65.172 18.8017 64.8628 19.4707C64.5536 20.1913 64.0898 20.8088 63.4714 21.272C62.8015 21.7867 62.0801 22.1984 61.2555 22.4558C60.328 22.6102 59.2973 22.7646 58.3182 22.7131ZM91.7623 22.3014H88.6189V9.89802C88.6704 9.02309 88.4128 8.19964 87.949 7.47911C87.5367 6.96445 86.8668 6.70712 85.8877 6.70712C85.4754 6.70712 85.0632 6.75858 84.7025 6.91298C84.2902 7.06738 83.878 7.22178 83.5172 7.47911C83.1565 7.73644 82.7958 7.99377 82.4351 8.30257C82.1774 8.5599 81.9713 8.81723 81.7652 9.12603V22.3014H78.5702V9.89802C78.6217 9.02309 78.3641 8.19964 77.9003 7.47911C77.488 6.96445 76.8181 6.70712 75.839 6.70712C75.4268 6.70712 75.066 6.75858 74.7053 6.91298C74.2931 7.06738 73.8808 7.27325 73.5201 7.47911C73.1078 7.68498 72.7986 7.99377 72.4379 8.30257C72.1803 8.5599 71.9741 8.81723 71.7165 9.12603V22.3014H68.573V4.18527H71.7165V5.67779C72.0257 5.42046 72.3864 5.16313 72.7471 4.95727C73.314 4.597 73.8808 4.33967 74.4992 4.13381C75.1176 3.92794 75.839 3.77354 76.5089 3.77354C77.1788 3.77354 77.8488 3.87648 78.4671 4.08234C79.034 4.28821 79.5493 4.54554 79.9616 4.9058C80.3738 5.2146 80.7345 5.62633 81.0437 6.08952L81.1468 6.29539C81.2498 6.19245 81.3529 6.08952 81.456 5.98659C81.9198 5.57486 82.3835 5.2146 82.8989 4.9058C83.4142 4.597 84.0326 4.33967 84.5994 4.13381C85.2178 3.92794 85.8877 3.82501 86.5576 3.82501C90.0102 3.82501 91.8138 5.83219 91.8138 9.58922L91.7623 22.3014ZM160.351 22.3014H157.208V4.18527H160.351V5.98659C160.815 5.57486 161.279 5.16313 161.845 4.85433C162.876 4.18527 164.113 3.82501 165.401 3.82501H166.071V7.32471L165.247 7.06738C165.143 7.06738 165.092 7.01591 164.989 7.01591H164.525C164.164 7.01591 163.752 7.11885 163.391 7.22178C162.928 7.37618 162.515 7.58204 162.103 7.83937C161.691 8.0967 161.33 8.4055 161.021 8.7143C160.763 8.97163 160.557 9.22896 160.403 9.48629L160.351 22.3014ZM116.601 22.3014H113.457V4.18527H116.601V5.98659C117.064 5.57486 117.528 5.16313 118.095 4.80287C119.126 4.13381 120.362 3.77354 121.651 3.77354H122.321V7.32471L121.496 7.11885C121.393 7.11885 121.342 7.06738 121.238 7.06738H120.723C120.311 7.06738 119.95 7.17031 119.589 7.27324C118.713 7.58204 117.889 8.0967 117.219 8.76577C116.961 9.0231 116.755 9.28043 116.601 9.58922V22.3014ZM102.275 12.986C101.193 12.9345 100.162 13.2948 99.389 14.0153C98.7191 14.6844 98.3068 15.5593 98.3584 16.5372C98.3068 17.4635 98.7191 18.3899 99.389 19.0075C100.162 19.7281 101.193 20.0883 102.275 20.0369C103.151 20.0369 104.027 19.8825 104.851 19.5222C105.573 19.2134 106.191 18.7502 106.707 18.1841V14.8902C106.191 14.3241 105.573 13.8094 104.851 13.5007C104.027 13.1919 103.151 12.986 102.275 12.986ZM139.584 11.7508H150.405C150.354 11.2876 150.251 10.8244 150.096 10.4127C149.89 9.74362 149.529 9.12603 149.066 8.5599C148.602 7.99377 148.035 7.53058 147.365 7.22178C146.592 6.86152 145.768 6.65565 144.943 6.70712C144.17 6.70712 143.346 6.86152 142.624 7.22178C141.954 7.58204 141.387 7.99377 140.924 8.5599C140.46 9.12603 140.099 9.74362 139.893 10.4127C139.79 10.8244 139.635 11.2876 139.584 11.7508Z" fill="#F90070"/>
<path d="M7.86885 29.1464C7.19894 29.1464 6.52903 29.0949 5.85911 29.0434C5.24073 28.992 4.62235 28.8376 4.00397 28.5802C3.43713 28.3744 2.87028 28.0656 2.35496 27.7053C1.78811 27.2936 1.27279 26.8819 0.809008 26.3672L0.448286 26.0069L2.30343 23.4851L2.81875 24.1027C3.38559 24.8232 4.10704 25.3893 4.98308 25.7496C5.91065 26.1099 6.88975 26.2643 7.92038 26.2128C8.59029 26.2128 9.26021 26.1099 9.93012 25.9555C10.497 25.8011 11.0638 25.4923 11.5276 25.132C11.9914 24.7203 12.3521 24.2571 12.6098 23.691C12.919 23.0219 13.022 22.2499 13.022 21.4779V20.5515C12.5582 20.9633 12.0945 21.2721 11.5791 21.5808C10.4454 22.2499 9.15714 22.6102 7.81732 22.6102C6.68362 22.6102 5.54992 22.4043 4.46776 21.9926C3.43712 21.5294 2.56109 20.9118 1.83964 20.0883C1.1182 19.2134 0.551349 18.1841 0.190627 17.1033C-0.221626 15.8681 -0.376221 14.5814 -0.376221 13.2433C-0.376221 11.9567 -0.221626 10.67 0.190627 9.43483C0.551349 8.35404 1.1182 7.32471 1.83964 6.44979C2.56109 5.62633 3.43712 4.95727 4.46776 4.49407C5.49839 4.03088 6.68362 3.77355 7.81732 3.77355C9.10561 3.77355 10.3424 4.08234 11.4761 4.69994C12.0429 5.00873 12.5582 5.369 13.022 5.83219V4.23674H16.1655V21.5294C16.217 22.7646 15.9593 23.9483 15.4955 25.0805C15.0833 26.0069 14.4649 26.8304 13.6404 27.4995C12.8674 28.1171 11.9399 28.5802 10.9608 28.8376C9.93012 28.992 8.89949 29.1464 7.86885 29.1464ZM44.714 22.7131C43.5803 22.816 42.4982 22.3528 41.7252 21.5294C41.0553 20.6545 40.6945 19.5222 40.7461 18.3899V7.17031H37.9634V4.23674H40.7461V-0.34375H43.941V4.23674H47.3421V7.17031H43.941V18.1841C43.941 18.6473 44.0441 19.059 44.2502 19.4707C44.4048 19.7281 44.714 19.831 45.0232 19.831C45.2809 19.831 45.5901 19.7795 45.8477 19.6251C46.0538 19.5222 46.26 19.3678 46.4146 19.2134L47.0845 18.4929L48.4243 21.1691L48.0636 21.5294C47.6513 21.8896 47.1875 22.1984 46.7238 22.4558C46.1054 22.6616 45.3839 22.7646 44.714 22.7131ZM28.2754 22.7131C27.0386 22.7131 25.8019 22.4558 24.6167 21.9926C23.5345 21.5294 22.5554 20.8603 21.7309 20.0369C20.9064 19.1619 20.288 18.1326 19.8757 17.0004C18.9997 14.5814 18.9997 11.9567 19.8757 9.53776C20.288 8.45697 20.9064 7.42764 21.6793 6.55272C22.4523 5.67779 23.3799 5.00873 24.4621 4.49407C25.5442 3.97941 26.781 3.72208 27.9662 3.72208C29.2545 3.72208 30.4913 3.97941 31.625 4.49407C32.6556 5.00873 33.5832 5.72926 34.3561 6.60419C35.0776 7.53058 35.6444 8.5599 36.0052 9.64069C36.3659 10.8244 36.572 12.0081 36.572 13.2433V14.3756H22.5554C22.6584 14.9417 22.7615 15.5078 22.9676 16.074C23.2253 16.7945 23.6376 17.4636 24.1529 18.0297C24.6682 18.5958 25.2866 19.0075 25.9565 19.3163C26.7295 19.6251 27.554 19.7795 28.3785 19.7795C29.306 19.7795 30.2336 19.6251 31.1096 19.2649C31.9857 18.9046 32.7587 18.3385 33.377 17.6694L33.8924 17.1033L35.6444 19.3163L35.2322 19.7281C34.3562 20.6545 33.3255 21.375 32.1403 21.8896C30.9551 22.5072 29.6152 22.7646 28.2754 22.7131ZM5.91065 19.2649C6.58056 19.5737 7.35353 19.7281 8.12651 19.7281C8.64183 19.7281 9.15714 19.6766 9.62093 19.5222C10.1362 19.3678 10.6 19.1619 11.0638 18.9046C11.4761 18.6473 11.8883 18.3899 12.249 18.0297C12.5067 17.7724 12.7644 17.4636 13.022 17.1033V9.33189C12.8159 9.0231 12.5582 8.7143 12.249 8.4055C11.8883 8.04524 11.5276 7.78791 11.0638 7.53058C10.6 7.27325 10.1362 7.06738 9.62093 6.91298C8.38417 6.55272 7.09588 6.65565 5.91065 7.17031C5.29227 7.47911 4.72542 7.94231 4.31316 8.50844C3.84938 9.12603 3.48866 9.84656 3.28253 10.5671C2.76721 12.2655 2.76721 14.1182 3.28253 15.8681C3.48866 16.5886 3.84938 17.3092 4.31316 17.9267C4.72542 18.4929 5.29227 18.9561 5.91065 19.2649ZM22.5554 11.7508H33.377C33.3255 11.2876 33.2224 10.8244 33.0679 10.4127C32.8617 9.74362 32.501 9.12603 32.0372 8.5599C31.5734 7.99377 31.0066 7.53058 30.3367 7.22178C29.5637 6.86152 28.7392 6.65565 27.9147 6.70712C27.1417 6.70712 26.3172 6.86152 25.5958 7.22178C24.9258 7.53058 24.359 7.99377 23.8952 8.5599C23.4314 9.12603 23.0707 9.74362 22.8646 10.4127C22.71 10.8244 22.6069 11.2876 22.5554 11.7508Z" fill="black"/>
<path d="M174.213 13.0889C173.389 13.0889 172.564 12.883 171.843 12.4713C169.678 11.1846 168.957 8.40547 170.194 6.19242C171.482 3.97938 174.265 3.20738 176.481 4.49404C178.696 5.78069 179.469 8.55986 178.181 10.7729C177.357 12.1625 175.862 13.0889 174.213 13.0889ZM174.213 4.64844C173.543 4.64844 172.873 4.80284 172.307 5.1631C171.74 5.52336 171.224 5.98656 170.915 6.55268C170.554 7.11881 170.4 7.78787 170.4 8.45693C170.4 9.84652 171.121 11.0817 172.307 11.8022C173.492 12.4713 174.935 12.4713 176.12 11.8022C176.687 11.442 177.202 10.9788 177.511 10.4126C177.872 9.84652 178.027 9.17746 178.027 8.5084C178.027 7.83934 177.872 7.17028 177.511 6.60415C177.151 6.03802 176.687 5.52336 176.12 5.21456C175.553 4.80283 174.883 4.64844 174.213 4.64844ZM176.223 10.5156H175.244V10.4641L174.265 8.8172H173.698V10.5156H172.873V6.08949H174.213C174.625 6.03802 175.089 6.19242 175.45 6.39829C175.914 6.86148 175.965 7.58201 175.656 8.09667C175.502 8.30253 175.347 8.45693 175.141 8.5084L176.223 10.5156ZM173.698 8.09667H174.162C174.368 8.09667 174.574 8.0452 174.729 7.89081C174.883 7.78787 174.935 7.63347 174.935 7.42761C174.935 7.27321 174.883 7.06735 174.78 6.96441C174.574 6.86148 174.368 6.81001 174.162 6.81001H173.698V8.09667Z" fill="black"/>
<path d="M98.3587 38.0496V40.8288H96.9158V33.2633H98.3587V33.9323C98.7709 33.4177 99.3893 33.1089 100.059 33.1089C101.451 33.1089 102.481 34.1382 102.481 35.991C102.481 37.8438 101.451 38.8731 100.059 38.8731C99.3378 38.9246 98.7194 38.5643 98.3587 38.0496ZM100.935 36.0425C100.935 35.1161 100.368 34.447 99.5439 34.447C99.0801 34.447 98.6163 34.7043 98.3071 35.0646V37.0203C98.6163 37.3806 99.0801 37.6379 99.5439 37.6379C100.42 37.6379 100.935 36.9688 100.935 36.0425Z" fill="black"/>
<path d="M103.151 36.0427C103.151 34.4988 104.233 33.1606 106.037 33.1606C107.841 33.1606 108.923 34.4988 108.923 36.0427C108.923 37.5867 107.841 38.9248 106.037 38.9248C104.233 38.9248 103.151 37.5867 103.151 36.0427ZM107.428 36.0427C107.428 35.1678 106.913 34.4473 106.037 34.4473C105.161 34.4473 104.646 35.1678 104.646 36.0427C104.646 36.9177 105.161 37.6382 106.037 37.6382C106.913 37.6382 107.428 36.8662 107.428 36.0427Z" fill="black"/>
<path d="M114.797 38.7704L113.664 35.0648L112.53 38.7704H110.984L109.335 33.3149H110.829L111.86 36.969L113.045 33.3149H114.334L115.57 36.969L116.601 33.3149H118.095L116.446 38.7704H114.797Z" fill="black"/>
<path d="M118.353 36.0427C118.301 34.4988 119.538 33.2121 121.033 33.1606C121.084 33.1606 121.136 33.1606 121.187 33.1606C122.836 33.1606 123.918 34.3958 123.918 36.1971V36.5059H119.847C119.95 37.175 120.517 37.7926 121.445 37.7926C121.96 37.7926 122.475 37.5867 122.888 37.2265L123.506 38.1529C122.939 38.6675 122.115 38.9248 121.239 38.9248C119.641 38.9248 118.353 37.7926 118.353 36.0427ZM121.187 34.2929C120.466 34.2929 119.899 34.8076 119.847 35.5281H122.527C122.527 34.8076 121.909 34.2414 121.187 34.2929C121.239 34.2929 121.239 34.2929 121.187 34.2929Z" fill="black"/>
<path d="M125 38.7703V33.3148H126.443V34.0868C126.856 33.5722 127.526 33.2119 128.195 33.2119V34.6015C128.041 34.6015 127.938 34.55 127.783 34.55C127.319 34.55 126.649 34.8074 126.443 35.1676V38.7703H125Z" fill="black"/>
<path d="M128.711 36.0427C128.659 34.4988 129.896 33.2121 131.391 33.1606C131.442 33.1606 131.494 33.1606 131.545 33.1606C133.194 33.1606 134.276 34.3958 134.276 36.1971V36.5059H130.205C130.309 37.175 130.875 37.7926 131.803 37.7926C132.318 37.7926 132.834 37.5867 133.246 37.2265L133.864 38.1529C133.297 38.6675 132.473 38.9248 131.597 38.9248C129.999 38.9248 128.711 37.7926 128.711 36.0427ZM131.545 34.2929C130.824 34.2929 130.257 34.8076 130.205 35.5281H132.885C132.885 34.8076 132.267 34.2414 131.545 34.2929C131.597 34.2929 131.545 34.2929 131.545 34.2929Z" fill="black"/>
<path d="M139.12 38.7703V38.0498C138.708 38.5645 138.09 38.8733 137.42 38.8733C136.028 38.8733 134.998 37.8439 134.998 35.9911C134.998 34.1898 136.028 33.109 137.42 33.109C138.09 33.109 138.708 33.4178 139.12 33.9325V31.1533H140.563V38.7189H139.12V38.7703ZM139.12 36.969V35.0133C138.811 34.6016 138.347 34.3957 137.884 34.3957C137.059 34.3957 136.492 35.0648 136.492 35.9911C136.492 36.9175 137.059 37.5866 137.884 37.5866C138.347 37.6381 138.811 37.3807 139.12 36.969Z" fill="black"/>
<path d="M144.892 38.7701V31.2046H146.334V33.9838C146.747 33.4691 147.365 33.1603 148.035 33.1603C149.426 33.1603 150.457 34.2411 150.457 36.0424C150.457 37.8437 149.426 38.9245 148.035 38.9245C147.365 38.9245 146.747 38.6157 146.334 38.1011V38.8216L144.892 38.7701ZM147.571 37.6379C148.396 37.6379 148.963 37.0203 148.963 36.0424C148.963 35.0646 148.396 34.447 147.571 34.447C147.107 34.447 146.644 34.7043 146.334 35.0646V37.0203C146.644 37.3805 147.056 37.5864 147.571 37.6379Z" fill="black"/>
<path d="M151.539 39.594C151.694 39.6455 151.849 39.6969 152.003 39.6969C152.364 39.6969 152.621 39.594 152.776 39.2852L152.982 38.822L150.766 33.2637H152.312L153.755 37.0722L155.198 33.2637H156.744L154.167 39.6455C153.755 40.6748 153.034 40.9321 152.106 40.9836C151.849 40.9836 151.642 40.9321 151.385 40.8807L151.539 39.594Z" fill="black"/>
<path d="M171.532 45.9966C165.76 45.9966 161.081 41.3232 161.081 35.5584C161.081 29.7935 165.76 25.1201 171.532 25.1201C177.304 25.1201 181.984 29.7935 181.984 35.5584C181.984 41.3231 177.304 45.9966 171.532 45.9966ZM175.331 30.6651V37.3623C175.331 37.8648 174.923 38.2722 174.42 38.2722C173.917 38.2722 173.509 37.8648 173.509 37.3623V30.6651H171.231V37.2531C171.231 39.012 172.659 40.4379 174.42 40.4379C176.181 40.4379 177.609 39.012 177.609 37.2531V30.6651H175.331ZM171.962 39.7898C171.636 39.4639 171.38 39.0981 171.201 38.6755C171.135 38.5215 171.082 38.3633 171.039 38.2176H167.732C167.732 38.2176 168.716 37.3107 169.96 36.0655C170.581 35.4438 170.867 34.6121 170.867 33.7979V33.7927C170.867 32.9784 170.582 32.1645 169.96 31.5433C168.715 30.2995 166.709 30.3268 165.464 31.5706C164.841 32.1925 164.544 33.0309 164.544 33.8499H166.821C166.821 33.6133 166.91 33.3569 167.088 33.1792C167.444 32.8238 168.021 32.8101 168.377 33.1655C168.733 33.5209 168.733 34.0902 168.377 34.4456C168.377 34.4456 166.558 36.2035 164.973 37.7478C164.693 38.0204 164.543 38.4434 164.543 38.8646C164.543 39.749 165.261 40.4925 166.147 40.4925H172.906C172.557 40.3105 172.24 40.0672 171.962 39.7898ZM177.705 39.737V39.5099H177.874C177.964 39.5099 178.015 39.5491 178.015 39.6234C178.015 39.69 177.964 39.737 177.874 39.737H177.705ZM177.572 40.0855H177.705V39.8466H177.85L178.015 40.0855H178.176L177.991 39.8231C178.085 39.7957 178.152 39.7252 178.152 39.6116C178.152 39.4668 178.038 39.3923 177.885 39.3923H177.572V40.0855H177.572ZM178.442 39.7527C178.442 40.0855 178.179 40.3636 177.838 40.3636C177.493 40.3636 177.235 40.0894 177.235 39.7566V39.7527C177.235 39.4198 177.497 39.1418 177.838 39.1418C178.183 39.1418 178.442 39.4158 178.442 39.7488V39.7527ZM177.156 39.7565C177.156 40.1285 177.454 40.4379 177.838 40.4379C178.219 40.4379 178.521 40.1247 178.521 39.7526V39.7487C178.521 39.3767 178.223 39.0672 177.838 39.0672C177.458 39.0672 177.156 39.3806 177.156 39.7526V39.7565Z" fill="black"/>
</g>
<defs>
<clipPath id="clip0_54_66849">
<rect width="182" height="47" fill="white"/>
</clipPath>
</defs>
</svg>
