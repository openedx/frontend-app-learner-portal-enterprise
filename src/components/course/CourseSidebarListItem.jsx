import React from 'react';
import PropTypes from 'prop-types';
import { Col, Icon, Row } from '@openedx/paragon';

const CourseSidebarListItem = ({
  icon,
  label,
  content,
}) => (
  <Row as="li" className="d-flex align-items-start border-bottom py-3">
    <Col className="d-flex align-items-center">
      <Icon className="mr-3" src={icon} />
      <span>{label}:</span>
    </Col>
    <Col>
      {content}
    </Col>
  </Row>
);

CourseSidebarListItem.propTypes = {
  icon: PropTypes.shape().isRequired,
  label: PropTypes.string.isRequired,
  content: PropTypes.node.isRequired,
};

export default CourseSidebarListItem;
