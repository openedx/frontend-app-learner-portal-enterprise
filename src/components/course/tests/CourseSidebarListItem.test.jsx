import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import { Person } from '@openedx/paragon/icons';

import CourseSidebarListItem from '../CourseSidebarListItem';

describe('CourseSidebarListItem', () => {
  it('renders', () => {
    const props = {
      icon: Person,
      label: 'Test label',
      content: <div data-testid="test-content" />,
    };
    const tree = renderer
      .create(<CourseSidebarListItem {...props} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
