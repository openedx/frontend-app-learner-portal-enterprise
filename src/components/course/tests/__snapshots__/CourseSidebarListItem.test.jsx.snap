// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CourseSidebarListItem renders 1`] = `
<li
  className="d-flex align-items-start border-bottom py-3 row"
>
  <div
    className="d-flex align-items-center col"
  >
    <span
      className="pgn__icon mr-3"
    >
      <svg
        aria-hidden={true}
        fill="none"
        focusable={false}
        height={24}
        role="img"
        viewBox="0 0 24 24"
        width={24}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4Zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4Z"
          fill="currentColor"
        />
      </svg>
    </span>
    <span>
      Test label
      :
    </span>
  </div>
  <div
    className="col"
  >
    <div
      data-testid="test-content"
    />
  </div>
</li>
`;
